recurly/__init__.py

```diff
@@ -265,6 +266,13 @@ def __getattr__(self, name):
                 raise AttributeError(name)
             resp, elem = BillingInfo.element_for_url(billing_info_url)
             return BillingInfo.from_element(elem)
+        if name == 'parent_account':
+            try:
+                parent_account_url = self._elem.find('parent_account').attrib['href']
+            except (AttributeError, KeyError):
+                raise AttributeError(name)
```

Comment:
This throws an exception when trying to access `parent_account` on an account that has no parent... Should we just return `null` instead of throwing an error?

Reply:
I don't think we need this code. Shouldn't this be handled by the relatiator?

Reply:
This can be handled by the relatiator, but I didn't realize it because you need to call the function `parent_account()` rather than reading the attribute `parent_account`. So to get the parent account code, you would do this
```python
acct.parent_account().account_code
```
rather than 
```python
acct.parent_account.account_code
```

For the sake of simplicity and using the tools the library gives us already, I'll go ahead and remove it and update the spec and the scripts in the conversation portion of this PR

Reply:
Right, all links are turned into functions.